esky/bdist_esky/f_py2app.py

```diff
@@ -110,30 +109,57 @@ def copy_to_bootstrap_env(src,dst=None):
                 #  have no interest in examining right now.  Eventually this
                 #  guard will be more conservative.
                 pass
+        bsdir = dist.bootstrap_dir
         copy_to_bootstrap_env("Contents/Resources/include")
         if sys.version_info[:2] < (3, 3):
             copy_to_bootstrap_env("Contents/Resources/lib/"+pydir+"/config")
         else:
             copy_to_bootstrap_env("Contents/Resources/lib/"+pydir+"/config-%d.%dm"
                                    % sys.version_info[:2])
+            # copy across the zip file that we need to run the boostrap application
+            # from the inner package.
+            python_name = 'python%d%d' % sys.version_info[:2]
+            bs_lib_dir = os.path.join('Contents', 'Resources', 'lib')
+            zip_name = os.path.join(bs_lib_dir, '{}.zip'.format(python_name))
+            copy_to_bootstrap_env(zip_name)
```

Comment:
This is only being used temporarily,  in other parts of the project we do this.
`import tempfile`
 `with tempfile.mkdtemp() as tdir:`
do all temp stuff in this folder